main.go

```diff
@@ -116,8 +116,48 @@ func (f *file) selectorExprToSchema(selectorType *ast.SelectorExpr, doc string,
 	pkgAlias := selectorType.X.(*ast.Ident).Name
 	typeName := selectorType.Sel.Name
 
-	// TODO(mengqiy): handle special cases here.
-	// e.g. IntOrString https://github.com/kubernetes/kubernetes/blob/a3ccea9d8743f2ff82e41b6c2af6dc2c41dc7b10/staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go#L41-L45
+	typ := TypeReference{
+		TypeName:    typeName,
+		PackageName: f.importPaths[pkgAlias],
+	}
+
+	time := TypeReference{TypeName: "Time", PackageName: "k8s.io/apimachinery/pkg/apis/meta/v1"}
```

Comment:
This was one of the issues that I hit when generating knative schema.
I had to manually change it. This is awesome!